index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <title>Fork Me! FCC: Test Suite Template</title>
+        <link rel="stylesheet" href="css/style.css" />
+    </head>
+
+    <body>
+        <h1 id="title">freeCodeCamp</h1>
+        <p id="description">
+            Thank you for taking the time to help us improve the platform
+        </p>
+
+        <form id="survey-form">
+            <label id="name-label">Name</label><br />
+            <input
+                class="stretched"
+                type="text"
+                id="name"
+                placeholder="Enter Your Name"
+                required
+            /><br />
+            <label id="email-label">Email</label><br />
+
+            <input
+                class="stretched"
```

Comment:
Please use an indention of two lines that is [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)